src/contract_client/utils.ts

```diff
@@ -1,7 +1,11 @@
+// @ts-ignore
+import * as jsxdr from "@stellar/js-xdr";
+import { xdr } from "@stellar/stellar-base";
```

Comment:
Using your own instance is better, I think?

```suggestion
import { xdr } from "..";
```